plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -5276,32 +5276,6 @@ private void testParquetDictionaryPredicatePushdown(Session session)
         assertNoDataRead("SELECT * FROM " + tableName + " WHERE n = 3");
     }
 
```

Comment:
Shorten commit message to `Move testParquetOnlyNullsRowGroupPruning to io.trino.testing`

Reply:
done.